CONTRIBUTING.md

```diff
@@ -104,9 +104,31 @@ or alternatively `yarn buildall` to build all of them. Since most apps have
 dependencies on `mlchartlib` it makes sense to run `yarn buildall` at least
 once.
 
-Tests are in the `apps` directory. To execute them run `yarn e2eall`.
-Sometimes it is preferable to watch the execution and select only individual
-test cases. This is possible using `yarn e2e-watch`.
+### Testing
+
+#### Run e2e tests locally with mock data
+
+1. git clone <https://github.com/microsoft/responsible-ai-widget>
+2. `cd responsible-ai-widgets`
+3. `yarn install`
+4. `yarn build`
+5. To execute tests run `yarn e2eall`. Sometimes it is preferable to watch the execution and select only individual test cases. This is possible using `yarn e2e --watch`
+
+cypress window will open locally - select test file to run the tests
+
+#### Run e2e tests locally with notebook data
+
+1. git clone <https://github.com/microsoft/responsible-ai-widget>
+2. `cd responsible-ai-widgets` (It is recommended to create a new virtual environment and install the dependencies)
```

Comment:
Small nitpick: I think this needs to be `toolbox` rather than `widgets` due to the repo renaming.

Reply:
Great catch! fixed to toolbox.